src/v2/Components/Links/BackLink.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { ChevronIcon, Flex, Text } from "@artsy/palette"
+import { StyledLink } from "./StyledLink"
+
+interface BackLinkProps {
+  linkText: string
+  path: string
+}
+
+export const BackLink: React.FC<BackLinkProps> = ({ linkText, path }) => {
```

Comment:
This is for a link like `< Back to ...`

I don't _think_ we have anything like this in palette and we'll def be able to use it for the shows page too.